src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -97,11 +98,26 @@ public static void removeSpawnBiome(Biome biome)
     }
     */
 
+    /**
+     * Add biomes that you add to the overworld without using {@link BiomeManager#addBiome(BiomeType, BiomeEntry)}
+     */
+    public static void addAdditionalOverworldBiomes(RegistryKey<Biome> biome){
+        if (!"minecraft".equals(biome.func_240901_a_().getNamespace()) && additionalOverworldBiomes.stream().noneMatch(entry -> entry.func_240901_a_().equals(biome.func_240901_a_())))
```

Comment:
Unfortunately, `RegistryKey` does not support `equals`.

Reply:
That is true but I believe you can use `RegistryKey#func_244356_a` to compare them which is likely to make this line a bit cleaner

Reply:
That method does something weird/different. It compares the registry name/id with a registry key of a registry, I think?

Reply:
Ah you are correct I misread which resource locations that method compares.